modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/groups.ftl

```diff
@@ -32,7 +32,11 @@
 		_parentDLFolderId = 0
 	/>
 
-	<#assign homePageContentLayoutModels = dataFactory.newContentPageLayoutModels(groupId, "home") />
+	<#assign homePageContentLayoutModels = dataFactory.newContentPageLayoutModels(groupId, "home")
```

Comment:
@slnn Please SF the indents and new lines in the ftl.

Reply:
Done